locales/en/translation.json

```
@@ -88,7 +88,7 @@
         "reload": "Reload",
         "select-all": "Select All",
         "send-feedback": "Send feedback",
-        "should-migrate-to-flexible": "Your flexible calendar is empty. Do you want to start by migrating the existing fixed calendar database to your flexible one?",
+        "should-migrate-to-flexible": "One of the new features of this Time to Leave release is a flexible number of entries for the calendar.\n\nThis new view uses a new database format.\nTo use it, you can either migrate your existing database to the new format or make a fresh start. Please choose one of the options below.",
```

I think the "Choose one..." is unnecessary.

```suggestion
        "should-migrate-to-flexible": "One of the new features of this Time to Leave release is a flexible number of entries for the calendar.\n\nThis new view uses a new database format.\nTo use it, you can either migrate your existing database to the new format or make a fresh start.",
```